community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/command/PhysicalLogNeoCommandReaderV2_1.java

```diff
@@ -309,7 +310,7 @@ public boolean visitRelationshipGroupCommand( Command.RelationshipGroupCommand c
             {
                 throw new IOException( "Illegal in use flag: " + inUseByte );
             }
-            int type = channel.getShort();
+            int type = shortToUnsignedInt( channel.getShort() );
```

Comment:
How it's possible without an upgrade to get a code that use different version of command reader?